### PR TITLE
Resolve Merge Conflicts With Upstream

### PR DESCRIPTION
**Pull request by drewjj**
# PR Details

## Description

This resolves merge conflicts with upstream develop. This really only includes changes made by the WYDOT fork for the firmware manager service.

## How Has This Been Tested?

Unit tests.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [x] My changes require updates and/or additions to the unit tests:
  - [x] I have modified/added tests to cover my changes.
- [x] All existing tests pass.

### CODE REVIEW

**Review by dmccoystephenson — APPROVED** [review]
Looks good & the unit tests pass!

File: services/addons/images/firmware_manager/sample.env [hunk 1]
```diff
@@ -7,9 +7,13 @@ PG_DB_NAME=""
 PG_DB_USER=""
 PG_DB_PASS=""
 
-# Blob storage variables
-BLOB_STORAGE_PROVIDER="GCP"
-BLOB_STORAGE_BUCKET=""
+# Blob storage variables (only 'GCP' and 'DOCKER' are supported at this time)
+BLOB_STORAGE_PROVIDER=DOCKER
+## GCP Project and Bucket for BLOB storage (if using GCP)
+GCP_PROJECT=
```
Comment (mwodahl) [inline services/addons/images/firmware_manager/sample.env]: Looks like this is a duplicate of the GCP_PROJECT variable on line 19

Comment (drewjj) [reply services/addons/images/firmware_manager/sample.env]: I have removed this duplicate variable now.

**Review by mwodahl — APPROVED** [review]
Looks good to me!

**Review by dmccoystephenson — APPROVED** [review]

**Pull request merged**